azure-functions-powershell-library/BindingInformation.cs

```diff
@@ -0,0 +1,22 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+namespace AzureFunctions.PowerShell.SDK
+{
+    public class BindingInformation
+    {
+        public enum Directions
+        {
+            In = 0,
+            Out = 1, 
+            Inout = 2
```

Comment:
For a second I thought `Inout` was a mispelling of `Input`, but now I suspect this is probably for input-and-output. Should we consider renaming to `InOut`?

Reply:
I followed convention from the RPC metadata in the host/worker, which uses Inout. Can change it if it seems better that way, though

Reply:
We should check to see what the Python team is using for input-and-output.

Reply:
No strong feelings on this one. Agreed on checking in with Python team